command/agent/agent.go

```diff
@@ -1039,12 +1039,17 @@ func (a *Agent) EnableServiceMaintenance(serviceID string) error {
 		return nil
 	}
 
+	// Use default notes if no reason provided
+	if reason == "" {
+		reason = "Maintenance mode is enabled for this service"
```

Comment:
Should we make this a constant like `DefaultReason`. I also think we should have some text like "no reason provided" somewhere in there, maybe in parenthesis?